airbyte-webapp/src/components/LoadingBackdrop/LoadingBackdrop.tsx

```diff
@@ -0,0 +1,24 @@
+import React from "react";
+
+import Spinner from "../Spinner/Spinner";
+import styles from "./LoadingBackdrop.module.scss";
+
+interface LoadingBackdropProps {
+  isActive: boolean;
```

Comment:
I feel like `loading` may be a better term?

Reply:
Hm, yes, probably... Will fix it